packages/strapi-design-system/src/Badge/Badge.js

```diff
@@ -1,37 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
 import { Flex } from '../Flex';
 import { Typography } from '../Typography';
 
+const Base = styled(Flex)`
+  border-radius: 2px;
+  height: ${16 / 16}rem;
```

Comment:
seem's unnecessary? couldn't this just be `1rem`? 🤔

Reply:
I agree, but we always define values as px because that's what Figma displays. It avoids having the developer calculate px from rem values, when somebody needs to read this part.

Reply:
_hm_ wider discussion could be making `1rem === 10px` so it's actually a lot easier to work with....